src/plugins/console/server/lib/spec_definitions/json/generated/cluster.get_component_template.json

```diff
@@ -12,6 +11,6 @@
       "_component_template",
       "_component_template/{name}"
     ],
-    "documentation": "https://www.elastic.co/guide/en/elasticsearch/reference/master/getting-component-templates.html"
+    "documentation": "https://www.elastic.co/guide/en/elasticsearch/reference/master/indices-component-template.html"
   }
```

Comment:
Could we revert the docs url change? The old url is the correct one for "Get component template", the new one is for "Create or update". It is wrong in the ES repo I think.